test/e2e/storage/volume_metrics.go

```diff
@@ -51,7 +56,7 @@ var _ = SIGDescribe("[Serial] Volume metrics", func() {
 
 		pvc = newClaim(test, ns, "default")
 		var err error
-		metricsGrabber, err = metrics.NewMetricsGrabber(c, nil, false, false, true, false, false)
+		metricsGrabber, err = metrics.NewMetricsGrabber(c, nil, true, false, true, false, false)
```

Comment:
Wondering why `glog()` is used in NewMetricsGrabber? From what I've seen it's much more common in the e2e tests to call `framework.Logf()`.

Reply:
where is the glog used?

Reply:
In `metrics.NewMetricsGrabber` (L65)[https://github.com/kubernetes/kubernetes/pull/52836#discussion_r140320122]